### ISSUE TITLE
[accessibility] Controls are missing `role="button"`

### ISSUE THREAD

**Issue by Malvoz** [davidchouse/Leaflet.NavBar#7]
The controls use `<a>` elements but behave like buttons, as such they should have [`role="button"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).

(Alternatively use `<button type="button">`.)
